client/extensions/woocommerce/app/shipping/shipping-header.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import SectionHeader from 'components/section-header';
+
+class ShippingHeader extends Component {
+	render() {
+		const { label, description, children } = this.props;
+
+		const labelContent = (
+			<span>
+				<div className="shipping__header">{ label }</div>
```

Comment:
I am surprised to see a div ( a block element ) inside a span ( an inline element ) - I don't think that's proper HTML